scripts/cmake_setup.sh

```diff
@@ -9,7 +9,7 @@ set -e
 function print_help {
   echo "Usage: ./cmake_setup.sh <cmake_source_directory> <install_prefix>"
   echo
-  exit 1
+  return 1
```

Comment:
Will this change the behavior of the script? I think it may have been intended to early-exit when `print_help` is called.